File: InClass/OBJC_HelloWorld/HelloWorld/ViewController.m
//
//  ViewController.m
//  HelloWorld
//
//  Created by Blake Oistad on 9/21/15.
//  Copyright © 2015 Blake Oistad. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@property (nonatomic, weak) IBOutlet UILabel        *welcomeLabel;

@property (nonatomic, weak) IBOutlet UITextField    *nameTextField;

@property (nonatomic, strong)        UILabel        *testLabel;

@end

@implementation ViewController

#pragma mark - Interactivity Methods

- (IBAction)goButtonPressed:(id)sender {
    NSLog(@"ButtonPressed %@",_nameTextField.text);
    _welcomeLabel.text = [NSString stringWithFormat:@"Hello %@!",_nameTextField.text];
    _testLabel.text = @"I'm real!";
}

- (BOOL)textFieldShouldReturn:(UITextField *)textField {
    [textField resignFirstResponder];
    return true;
}

#pragma mark - Life Cycle Methods


- (void)viewDidLoad {
    [super viewDidLoad];
    
    _testLabel = [[UILabel alloc] initWithFrame:
                          CGRectMake(20.0, 200.0, 100.0, 21.0)];
    _testLabel.text = @"I'm a test label";
    [self.view addSubview:_testLabel];
    
    
    
    

}








- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];

}

@end

File: In Class/MyFirstSwift/MyFirstSwift/ViewController.swift
//
//  ViewController.swift
//  MyFirstSwift
//
//  Created by Blake Oistad on 10/26/15.
//  Copyright © 2015 Blake Oistad. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    
    
    @IBOutlet weak var myLabel   : UILabel!
    @IBOutlet weak var myButton  : UIButton!
    
    
    //MARK: - Interactivity Methods
    
    @IBAction func myButtonPressed(sender: UIButton) {
        print("My Button Pressed")
        myLabel.text = "Pressed"
        myButton.setTitle("Again", forState: .Normal)
    }
    
    //MARK: - Life Cycle Methods
    
    override func viewDidLoad() {
        super.viewDidLoad()

    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()

    }


}

